src/app/components/dialog/dialog.js

```diff
@@ -17,40 +17,40 @@ function PutAway({noTitle, onClick}) {
     );
 }
 
-export function FooterDialog({
-    isOpen, title, children, className
-}) {
-    React.useLayoutEffect(
-        () => {
-            const footerEl = document.getElementById('footer');
+export function FooterDialog({isOpen, title, children, className}) {
+    React.useLayoutEffect(() => {
+        const footerEl = document.getElementById('footer');
 
-            footerEl?.style.setProperty('z-index', 1);
+        footerEl?.style.setProperty('z-index', 1);
 
-            return () => footerEl?.style.removeProperty('z-index');
-        },
-        []
-    );
+        return () => footerEl?.style.removeProperty('z-index');
+    }, []);
 
     if (!isOpen) {
         return null;
     }
 
     return (
         <dialog className={cn('footer-dialog', className)}>
-            {
-                title &&
-                    <div className="title-bar">
-                        <RawHTML Tag="span" html={title} />
-                    </div>
-            }
+            {title && (
+                <div className="title-bar">
+                    <RawHTML Tag="span" html={title} />
+                </div>
+            )}
             {children}
         </dialog>
     );
 }
 
+// eslint-disable-next-line complexity
```

Comment:
I was looking at the complexity here, and ran prettier on the file, so almost all the changes are just reformatting.